### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by Biksco** [aclap-dev/video-downloadhelper#1046] [License & Purchase requests]
### What's your issue?

Video download no longer works on Youtube videos with OSX Chrome. (This is why I bought it) Like a refund request so I can use it on Firefox.

### The 10 first digits (only 10) of your license number or email address

e58e4e5d4f - eric@*****guy.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by wagner521** [comment]
estou com o msm problema :/

**Comment by jcfrog** [comment]
> Video download no longer works on Youtube videos with OSX Chrome. (This is why I bought it) Like a refund request so I can use it on Firefox.

Hi,
Sorry, Chrome extension never allowed Youtube downloads. 
Refund is only available the first moth after purchase.
